core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -69,14 +69,17 @@ var CreatorDashboardPage = function() {
   // Returns all exploration card elements with the given name.
   var _getExplorationElements = async function(explorationTitle) {
     await waitFor.visibilityOf(explorationDashboardCard);
-    return await allExplorationCards.filter(async function(tile) {
-      var text = await tile.getText();
+    var filteredExplorationCards = [];
+    for (i = 0; i < allExplorationCards.length; i++) {
+      var text = action.getText(allExplorationCards[i]);
```

Comment:
I don't think this is a correct usage of `action.getText`. Here's the code:

https://github.com/oppia/oppia/blob/5d81029defe519017194864bf8a0586fdb511f3d/core/tests/protractor_utils/action.js#L54